gpkit/keydict.py

```diff
@@ -121,9 +123,8 @@ def __setitem__(self, key, value):
         key, idx = self.parse_and_index(key)
         if key not in self.keymap:
             self.keymap[key].add(key)
-            if hasattr(key, "keys") and self.keymapping:
-                for mapkey in key.keys:
-                    self.keymap[mapkey].add(key)
+            if hasattr(key, "keys"):
+                self._unmapped_keys.add(key)
```

Comment:
confirming: `__setitem__` is the only method that can add a key? There's no `add`, `update`, etc?